functions/openhab.bash

```diff
@@ -68,12 +68,6 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
   openhabVersion="$(apt-cache madison openhab2 | head -n 1 | cut -d'|' -f2 | xargs)"
 
   local APT_INST_OPTS="-y --allow-downgrades"
```

Comment:
It would appear that this variable is no longer necessary. please remove it and put the options in the command itself